maxbalves/PostBy
# [Sprint 4] Data Screen + Functionality to Delete Specific Data

## Description
- A new screen allows users to see all their likes, dislikes, comments, and posts.
- Users are able to delete specific likes, dislikes, comments, and posts by swiping to the left.

## Milestones
Milestone/Sprint 4, [#30 Implement screen to see posts, likes, dislikes](https://github.com/maxbalves/PostBy/issues/30)

## Test Plan
- Tested the deletion process with all data types (Likes/Dislikes/Comments/Posts) and various possibilities:
    - More than 10+ posts, delete 2, and try to add 2 more to see if they display
    - Check that infinite scrolling works even after deletion
    - Check if relations are deleted properly
    - Check if comments are deleted after a post is deleted
    - etc...

_Demo:_
<img src="http://g.recordit.co/zoHACONotX.gif" width=250px>

## Future Plans
- I hope to work on some of the UI in order to allow the app to be more user friendly. The goal is to improve the design of every screen, including the Data Screen just created.
